### DISCUSSION TITLE
Tell me something

### DISCUSSION THREAD

**Discussion by DavidHuber-NOAA** [DavidHuber-NOAA/UIFCW_demo#4] [General]
Tell me something about yourself.

- What's your favorite cloud type or weather phenomenon?
- Have you worked on a weather model and if so which one(s)?
- What's the weather like where you are today?
- What are you hoping to get out of this course and/or the UIFCW workshop?

**Comment by DavidHuber-NOAA** [comment]
My favorite cloud type is Cumulus Castelanus.

**Reply by gspetro-NOAA** [reply]
What are the other options...? I know little about clouds. :(

**Comment by AndrewHazelton** [comment]
I have worked on the Hurricane Analysis and Forecast System (HAFS).

**Comment by shreyasdhavale** [comment]
I have worked on the WRF model.

**Comment by jobate** [comment]
-Mammatus clouds and the ionization of atmospheric gases forming the auroras.
-No.
-Hot and beautiful.
-Learning skills and information that I had not previously had.

**Comment by kvenable2011** [comment]
I work with wildfire impacts on water quality.

**Comment by gspetro-NOAA** [comment]
Hi everyone! 
I have worked on documentation for the Short-Range Weather (SRW) Application, the UFS Weather Model, the Unified Post Processor (UPP), and the Land Data Assimilation (DA) System. :)

**Comment by Xuejin-Zhang** [comment]
My favorite weather phenomenon is hurricane. I work on HAFS.

**Comment by deltonwentworth** [comment]
My favorite cloud/weather phenomenon is the Asperitas clouds.

**Comment by WramstromMia** [comment]
My favorite cloud is pileus.

I have been working on the HAFS model.

**Comment by OUCAPS** [comment]
I have worked on FV3 model

**Comment by wshimel** [comment]
Tell me something about yourself.

What's your favorite cloud type or weather phenomenon?
I like fog.  I enjoy the how fog dampens temp and sound.

**Reply by kvenable2011** [reply]
I love precipitation.

**Comment by Travis-Hurson** [comment]
I used to work on jets, now I work on Jet.
